docs/api_walkthrough.rst

```diff
@@ -305,6 +305,40 @@ If the balances of the tokens involved for Alice and Bob are now checked they sh
 
 
 
-Interacting with the Raiden echo node
+Interacting with the Raiden Echo Node
 =====================================
-TODO once the echo node is ready
+For easy testing of Raiden, there is a specialized Raiden node running, the "Raiden Echo Node". The Echo Node responds to received transfers by sending a transfer back to the initiator. The echo transfer follows certain rules:
+
+- consecutive transfers with the same ``identifier`` and same ``amount`` are ignored
```

Comment:
This is not mentioned in the echo node [issue]( https://github.com/raiden-network/raiden/issues/651) and I guess by ignoring you mean that the echo node sends no echo amount back. Perhaps make it more explicit by replacing `ignored` with `sends no amount back`?